charged-ieee/lib.typ

```diff
@@ -40,11 +46,20 @@
   set figure(placement: top)
   show figure.where(kind: table): set figure.caption(position: top)
   show figure.where(kind: table): set text(size: 8pt)
-  show figure.caption.where(kind: table): smallcaps
-  show figure.where(kind: table): set figure(numbering: "I")
-
+  show figure.where(kind: table): set figure(supplement: [TABLE], numbering: "I")
```

Comment:
wouldn't it be a bit more appropriate to use `Table` here and then only transform to uppercase in a captions how rule? then, we could likely also skip the `set ref` with the supplement function (I'd like to get rid of those supplement functions entirely and deprecate them ideally.)